data/model/proxy_cache.py

```diff
@@ -0,0 +1,42 @@
+from data.database import (
+    ProxyCacheConfig,
+    User
+)
+from data.model.organization import get_organization
+
+
+def create_proxy_cache_config(
+    org_name,
+    upstream_registry,
+    upstream_registry_namespace,
+    upstream_registry_username,
+    upstream_registry_password,
+    staleness_period_s
+):
+    """
+        Creates proxy cache configuration for the given organization name
+    """
+    org = get_organization(org_name)
+    if not org:
+        return None
+
+    new_entry = ProxyCacheConfig.create(
+        user_id=org.id,
+        upstream_registry=upstream_registry,
+        upstream_registry_namespace=upstream_registry_namespace,
+        upstream_registry_username=upstream_registry_username,
+        upstream_registry_password=upstream_registry_password,
+        staleness_period_s=staleness_period_s
+    )
+    return new_entry
+
+
+def get_proxy_cache_config_for_org(org_name):
+    """
+        Return the Proxy-Cache-Config associated with the given organization name, or None if it doesn't exist.
+    """
+    org = get_organization(org_name)
+    if not org:
+        return None
```

Comment:
`get_organization` will throw an exception if no org is found with the given name, so we can remove this check.
We might want to catch the exception and throw a different one, like `ProxyCacheConfigDoesNotExist` or something.
What you think?

Reply:
I think we can remove the None check and throw the same exception raised by `get_organization`. Something like `ProxyCacheConfigDoesNotExist` might imply the proxy config does not exist. But, it is actually the org that does not exist because of which we are not able to create proxy config. 
 What would you say?

Reply:
That was my first thought too, but I think it might be useful to differentiate the errors by using different exceptions. 
With that said, we can go with your suggestion for now and change it if needed.

Reply:
Sure, sounds good.